test/e2e/runner/evidence.go

```diff
@@ -164,7 +175,7 @@ func generateLightClientAttackEvidence(
 
 	// add a new bogus validator and remove an existing one to
 	// vary the validator set slightly
-	pv, conflictingVals, err := mutateValidatorSet(ctx, privVals, vals)
+	pv, conflictingVals, err := mutateValidatorSet(ctx, privVals, vals, !validEvidence)
```

Comment:
If the `nop` parameter in `mutateValidatorSet` is a no-op, then maybe it would make sense to keep `mutateValidatorSet` as is and do (?)
```
if validEvidence {
    pv, conflictingVals, err := mutateValidatorSet(ctx, privVals, vals)
} else {
   // reuse privVals and vals
}
```